packages/data-portal-explore/src/components/FileTable.tsx

```diff
@@ -918,10 +929,17 @@ export class FileTable extends React.Component<IFileTableProps> {
         super(props);
         makeObservable(this);
 
-        this.columnVisibility = _.mapValues(
-            _.keyBy(this.defaultColumns, (c) => c.name),
-            () => true
+        const columnVisibilityMap = _.mapValues(
+            _.keyBy(
+                this.defaultColumns,
+                (c) => (c as IEnhancedDataTableColumn<Entity>).name
+            ),
+            (c) =>
+                typeof c !== 'number' &&
+                (c as IEnhancedDataTableColumn<Entity>).name !== 'Treatment'
```

Comment:
Let's try this instead
```suggestion
                !(c as IEnhancedDataTableColumn<Entity>).omit
```